lib/runtime/procs/source.js

```diff
@@ -22,17 +22,21 @@ var source = base.extend({
         var self = this;
 
         _.each(points, function(pt) {
-            if (timeField && !pt[timeField]) {
-                self.trigger('warning', self.runtime_error('RT-POINT-MISSING-TIME', {field: timeField}));
+            if (timeField) {
+                if (!pt[timeField]) {
+                    self.trigger('warning', self.runtime_error('RT-POINT-MISSING-TIME', {field: timeField}));
+                }
+                if (timeField !== 'time') {
+                    if (pt.time) {
+                        pt._time = pt.time;
```

Comment:
In talking this over with @dmajda this morning, we agreed that having this "magic" behavior was undesirable.

Instead the logic should be that if there is a field called `time` in the point and the `timeField` is not time (either undefined or something else), then we should throw a warning and remove `time` from the point.

To support the case where the user wants to keep that field but stash it in something else, then we should add a `-timeRenameField` option which would implement the logic to move the conflicting time into another field. I don't love the name of that option FWIW but I think it's the least bad that I could think of.